src/pages/de/blog/online-speisekarte/index.mdx

```diff
@@ -0,0 +1,92 @@
+
+### SEO Strategien mit einer online Speisekarte 
+
+Ein bisschen über dieses Thema zu sprechen habe ich schon angefangen, als ich die HTML Version einer online Speisekarte erwähnte. Das Ding ist, dass es mit einer PDF Version einer Speisekarte schwieriger sein wird, eine gute Platzierung in Suchmachinen bzw. Google zu erreichen.
+
+Durch eine HTML Version werden alle in Ihrer online Speisekarte erwähnten Inhalte zu Getränken, Speisen, Desserts, Angeboten, Ihrem Standort, Restaurantnamen usw. durch die von Google angewandte SEO Strategie für die Gäste leichter auffindbar.
+
+In den meisten Fällen wollen die Gäste, die nach einer online Speisekarte suchen, konsumieren und sich von einem Restaurant überraschen lassen. Seien Sie dieses Restaurant und erreichen Sie somit neue Gäste!
+
+### Online Speisekarte als Marketinginstrument
+
+Nehmen wir an, dass Ihr Konkurrent ein online Profil hat und bereits eine SEO freundliche online Speisekarte in seinem Profil hat... Die Warscheinlichkeit ist gross, dass Google Ihren Konkurrenten empfiehlt und nicht Ihren Betrieb.
+
+Erstellen Sie also ein Google Unternehmensprofil. Dort können Ihre Gäste Bewertungen über ihre Erfahrungen abgeben. Und vor allem: Füllen Sie die Seite mit so vielen Informationen wie möglich über Ihr Restaurant, einschliesslich Ihrer Online Speisekarte und Fotos. Die Art und Weise, wie Ihr Restaurant präsentiert wird, hat eine grosse Verantwortung für das Image Ihres Betriebs.
+
+Mit einer online Präsenz erhöhen Sie Ihre Chancen, gesehen zu werden, was automatisch mehr potenzielle Gäste generiert.
+
+### Ihre online Speisekarte als der zentrale Vertriebsweg
+
+Natürlich ist es wichtig, die Aufmerksamkeit auf Ihr Restaurant zu lenken, schliesslich haben Sie Zugang zu ponteziellen Gästen. Aber Sie haben ein klares Ziel: den Verkauf. Und wie kann Ihnen eine online Speisekarte dabei helfen? Ganz einfach.
+
+Denken Sie über die Struktur nach, wie Sie Ihre online Speisekarte entwickeln werden. Der erste Eindruck ist der, der bleibt, und in diesem Fall wird er durch Fotos und Produktbeschreibungen vermittelt. Klar sollte auch die Navigation durch die online Speisekarte klar und interaktiv sein.
+
+Bieten Sie Sonderangebote für Ihre online Speisekarte an, die oft ein entscheidender Faktor bei der Bestellung ist.
+
+Bieten Sie Leistungen an, die über das Restaurant vor Ort hinausgehen, denn die Gäste schätzen auch Schnelligkeit und Komfort. Wie wäre es mit einer exklusiven online Speisekarte für Take Away oder Delivery?
+
+Und natürlich... die Möglichkeit bieten, online zu bezahlen.
+
+Wenn ein potenzieller Gast sofort etwas zu essen sucht, ist die Wahrscheinlichkeit höher, dass er in den nächsten Minuten einen Kauf tätigt. Wenn Ihr Gast in Ihr Restaurant kommen muss, weiss er nicht mehr, was er essen wollte und er wird dann abgelenkt.
```

Comment:
"Wenn Ihr Gast in Ihr Restaurant kommen muss, weiss er nicht mehr, was er essen wollte und er wird dann abgelenkt." Dass ist zu wenig klar.